appveyor.yml

```diff
@@ -0,0 +1,24 @@
+image:
+- Visual Studio 2015
+platform: x64
+environment:
+ matrix:
+  - DC: dmd
+    DVersion: nightly
+    arch: x64
+  - DC: dmd
+    DVersion: nightly
+    arch: x86
+skip_tags: false
+build_script:
+ - ps: wget https://raw.githubusercontent.com/wilzbach/installer/appveyor-script/script/install.ps1 -OutFile install.ps1
```

Comment:
Can be made official once https://github.com/dlang/installer/pull/275 is merged, but works fine for now.